wilsonChan/assignments/mergeIntervals/lc56/lc56.py

```diff
@@ -0,0 +1,143 @@
+# Problem Statement #
+
+# Given a list of intervals, merge all the overlapping intervals to produce a list that has only mutually exclusive intervals.
+
+'''
+
```

Comment:
Nice pseudocode overview of your approach and thank you for the explanation of where you're running into issues with the test cases.